python/plugins/processing/tools/vector.py

```diff
@@ -69,6 +70,13 @@
     QVariant.Bool: "BOOLEAN"
 }
 
+TYPE_MAP_SPATIALITE_LAYER = {
+    QVariant.String: "VARCHAR",
+    QVariant.Double: "REAL",
+    QVariant.Int: "INTEGER",
+    QVariant.Bool: "INTEGER"
```

Comment:
IS this really like that?? or is it a typo? I am no expert in spatialite...so just asking, since it looks a bit confusing to me at first :-)